### PR TITLE
#CP-20 chore: support multiple languages 

### PR DESCRIPTION
**Pull request by Abdulkeza**
## What does this PR do?
Set up a project to use multiple languages

## Description of Task to be completed?
- [x]  Users should be able to change/select a language of their choice.
- [x] The site contents are presented in the newly set language.
- [x] At least, two languages are supported.  

## How should this be manually tested?
- [Clone](https://github.com/atlp-rwanda/phantom-be-codebandits.git)
- git co ch-support-multiple-languages-cp-20
- git pull
- npm start
- go to the postman
- set header to Accept-Language then value to "en" for english or "kn" for kinyarwanda or "fr" france
- send request

## Any background context you want to provide?
(N/A)
## What are the relevant pivotal tracker stories?
[amq27aTA](https://trello.com/c/amq27aTA/20-support-multiple-languages)

# Screenshots (if appropriate)
image
image
image
image

### CODE REVIEW

File: src/utils/i18n.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const __dirname = path.resolve();
+
+i18n.configure({
+	locales: ["en", "kn", "fr"],
+	defaultLocale: "en",
+	directory: `${__dirname}/locales/`,
+	headers: "Accept-Language"
+});
+
+export default i18n;
```
Comment (Yvad60) [inline src/utils/i18n.js]: for this configuration of i18n make sure we are using the same folder for all configurations to keep a consistent project structure pick a folder and use all configs there

File: package.json [hunk 2]
```diff
@@ -36,11 +36,15 @@
 		"*.{js,html,css,md}": "prettier --write"
 	},
 	"dependencies": {
+		"body-parser": "^1.19.2",
+		"eslint-config-prettier": "^8.5.0",
```
Comment (Yvad60) [inline package.json]: I think I forgot about that one, but everything related to eslint or prettier should be a dev dependency move that `eslint-config-prettier` in dev dependencies

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Good work with setting up translation it is working as expected in postman, take a quick look into the feedback below and try to fix your code climate issues to get your PR merged ASAP

File: src/utils/i18n.js (hunk 1 above)
Comment (Abdulkeza) [reply src/utils/i18n.js]: noted, thanks

File: package.json (hunk 2 above)
Comment (Abdulkeza) [reply package.json]: noted

**Review by feyton — DISMISSED** [review]
@Abdulkeza 
This is great work and the translation work seamlessly.

File: locales/en.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "welcome": "your welcome to Phatom",
```
Comment (samyshyaka) [inline locales/en.json]: There  is a simple typo with the welcome text, it should be "you are welcome to Phantom"

**Review by samyshyaka — CHANGES_REQUESTED** [review]
This is great work Abdul, It is working as expected. Just some small typos to fix, one being the name 'Phantom'.

**Review by Jkadhuwa — CHANGES_REQUESTED** [review]
Nice work @Abdulkeza , solve the eslint error on no_underscore_dangle

File: locales/en.json (hunk 3 above)
Comment (Abdulkeza) [reply locales/en.json]: Ohhhh!! okay

Comment (Abdulkeza) [reply locales/en.json]: Well solved

**Comment by Abdulkeza** [comment]
> Nice work @Abdulkeza , solve the eslint error on no_underscore_dangle

Well solved, thanks

**Review by samyshyaka — DISMISSED** [review]

**Review by Yvad60 — APPROVED** [review]

**Review by feyton — APPROVED** [review]
@Abdulkeza the issue I had has been resolved in this latest update and everything works well

**Review by Jkadhuwa — APPROVED** [review]

**Pull request merged**